docs/admin/external-auth.md

```diff
@@ -89,7 +89,7 @@ GitHub Enterprise requires the following environment variables:
 
 ```env
 CODER_EXTERNAL_AUTH_0_ID="primary-github"
-CODER_EXTERNAL_AUTH_0_TYPE=github-enterprise
```

Comment:
A quick code search shows that we are not using the type `github-enterprise` anywhere. So this change looks good to me.